### ISSUE TITLE
Invalid port number when loading a configuration file

### ISSUE THREAD

**Issue by jgeorgeson** [TigerVNC/tigervnc#583]
I saved a configuration file with vncviewer64-1.8.0.exe (confirmed SHA256 of local file vs bintray site), and opening the same file produces error "invalid port specified". I'm using an SSH tunnel (`ssh -L 5911:localhost:5901 server.example.com` because direct 5901 port is blocked on this server). However I have the same problem opening a config file for a direct connection (server2.example.com:1) to a server that allows it. Opening vncviewer64 and typing in the same servername works fine.

```
TigerVNC Configuration file Version 1.0



ServerName=localhost:11
X509CA=
X509CRL=
SecurityTypes=None,VncAuth,Plain,TLSNone,TLSVnc,TLSPlain,X509None,X509Vnc,X509Plain
DotWhenNoCursor=0
AutoSelect=1
FullColor=1
LowColorLevel=2
PreferredEncoding=Tight
CustomCompressLevel=0
CompressLevel=2
NoJPEG=0
QualityLevel=8
FullScreen=0
FullScreenAllMonitors=1
DesktopSize=1440x900
geometry=
RemoteResize=1
ViewOnly=0
Shared=0
AcceptClipboard=1
SendClipboard=1
MenuKey=F8
FullscreenSystemKeys=1
```

**Comment by CendioOssman** [comment]
I'm afraid I cannot reproduce the issue. The file quoted opens just fine. Is this perhaps a duplicate of #38?

**Comment by jgeorgeson** [comment]
I see, so I have to go through the GUI button clicks and dialogs to use a file. :( This can be closed as duplicate then. I'll go thumbs up and comment on #38. thanks.
